src/CameraProps.ts

```diff
@@ -38,6 +38,11 @@ export interface CameraProps extends ViewProps {
    * > Note: If you fully unmount the `<Camera>` component instead of using `isActive={false}`, the Camera will take a bit longer to start again. In return, it will use less resources since the Camera will be completely destroyed when unmounted.
    */
   isActive: boolean;
+  /**
+   * iOS Only. Uses a Metal preview `MTKView` instead of `AVCaptureVideoPreviewLayer` to present the camera preview.
+   * This is required for frame processor plugins that need to modify the preview before it is presented on screen
+   */
+  enableMetalPreview?: boolean;
```

Comment:
What's the overhead of this? Can we somehow remove that prop and make it automatic to detect whether we need a Metal preview or not? (whether the user wants to draw or not)

Reply:
Yeah this can probably be made automatic based on whether the user is doing any synchronous frame processing (the only time at which a Metal backed preview layer is required)